req-travis-conda.txt

```diff
@@ -10,3 +10,5 @@ pyzmq==14.7.0
 scipy==0.16.0
 six==1.9.0
 toolz==0.7.4
+Cython==0.23.4
```

Comment:
I don't really know, but Travis wasn't working without it. I'll try to investigate what happens here.